runtime/doc/lua.txt

```diff
@@ -830,6 +830,26 @@ vim.spell.check({str})                                      *vim.spell.check()*
                 "caps"  word should start with Capital
             - The position in {str} where the word begins.
 
+------------------------------------------------------------------------------
+VIM.PREVIEW						 	 *lua-preview*
+
+vim.preview.open_win({opts})			      *vim.preview.open_win()*
```

Comment:
Not sure `vim.preview` is the right name. "preview window" has a long established meaning in Vim (`:help preview-windows`)

I guess `vim.cmd_preview` might work, or `vim.inccommand`